### PR TITLE
Add bazzite

### PR DESCRIPTION
**Pull request by zen0bit**

### CODE REVIEW

**Review by flexiondotorg — CHANGES_REQUESTED** [review]
This pull request clatter bodhi.

I think it would also be wise to use `jq` to dynamically get the current release of Bazzite rather than hardcode release version. Bazzite release fairly often.

File: quickget
Comment (zen0bit) [inline quickget:1]: good to go

**Comment by zen0bit** [comment]
Tested with new release 2.0.0

**Comment by zen0bit** [comment]
ready

**Comment by zen0bit** [comment]
Tested with latest release...

**Comment by zen0bit** [comment]
continues in #892

**Pull request closed without merging**